spec/pubsub_spec.coffee

```diff
@@ -30,7 +30,6 @@ describe 'pubsub', ->
 
       robot.adapter.on 'connected', ->
         process.env.HUBOT_AUTH_ADMIN = '1'
-        robot.loadFile (path.resolve path.join 'node_modules/hubot/src/scripts'), 'auth.coffee'
```

Comment:
There is no such path `node_modules/hubot/src/scripts`, so I removed this line. Otherwise tests does not run